src/util/__fixtures__/server/index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
```

Comment:
This is a file mimicking SPA using React. DOM used here are extracted from real news website.